webapp/src/blocks.tsx

```diff
@@ -763,7 +774,9 @@ export class Editor extends srceditor.Editor {
         return blocklyOptions;
     }
 
+    private blocklyOptionsCache: Blockly.Options;
```

Comment:
Nitpick: move to top of file

Reply:
I wanted to keep the cache next to where it's used. I think that's a better paradigm since it only matters in that context.